### DISCUSSION TITLE
Mod items won't appear at merchant shop if you enable that mod mid campaign

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15413] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

.

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've fixed this in our private repo now, and the fix will be included in the next unstable build and the upcoming release.
